Source/ZoomNet/Models/ReportMeetingParticipant.cs

```diff
@@ -0,0 +1,20 @@
+using Newtonsoft.Json;
+using System;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Metrics of a participant.
+	/// </summary>
+	public class ReportMeetingParticipant : ReportParticipant
+	{
+		/// <summary>
+		/// Gets or sets the RegistrantID of the participant.
+		/// </summary>
+		/// <value>
+		/// The RegistrantID of the participant.  Only returned if registrant_id is included in the include_fields query parameter.
+		/// </value>
+		[JsonProperty(PropertyName = "registrant_id")]
+		public string RegistrantID { get; set; }
```

Comment:
```suggestion
		public string RegistrantId { get; set; }
```

Reply:
I made that change.  I would like to add the other endpoints to the reporting area, but those may come a little slower.  I just started a class, work full-time, and have two young kids so I will try to squeeze more in when I can but can't commit to anything yet.

Reply:
No prob. I'll merge this PR and whenever you have time to work on the other endpoint, simply submit another PR.